tests/Integration/Http/ValidationResponseTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Tempest\Integration\Http;
+
+use App\Controllers\ValidationController;
+use Tempest\Testing\IntegrationTest;
+use function Tempest\uri;
+
+/**
+ * @internal
```

Comment:
@aidan-casey what are these annotations?

Reply:
I added those in #198. They can be reverted just as easily if you don't like them.

`@internal` is technically not a huge deal anymore but basically says that it's not a class with a public API that should be used (helps IDE).

The size (`@small`, etc.) indicates how long the test should run to PHPunit and will fail if it goes over that threshold. I added after some seg fault errors that just hung up tests for a while.